scripts/Dockerfile.test

```diff
@@ -0,0 +1,47 @@
+# Container image for running the Ceph-CSI tests
```

Comment:
LGTM. I have a question
why we need containerized tests in centos CI?
can we run all static tests in Travis CI and run only time-consuming tests ie E2E in centos CI?

Reply:
It is that, or installing all dependencies on the CICO_NODE. Using the container image is just easier.

Reply:
Also, changes to the container image in the master branch can easily be backported to the ci/centos branch. If there is a separate script, we need to maintain is differently.